packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -39,6 +39,7 @@ declare global {
     lng?: number,
     slug?: string,
     sortDrafts?: string,
+    sortDraftsBy?: string,
```

Comment:
Having these two similarly-named fields is trouble, and I think in several places you get confused between them and that breaks things.